docs/modules/ROOT/pages/documentation/ecosystem/docker-server-usage.adoc

```diff
@@ -0,0 +1,173 @@
+== Usage
+
+=== Quick start
+
+To boot the default domain with HTTP listener exported on port 8080:
+
+....
+docker run -p 8080:8080 payara/server-full
+....
+
+The Docker container specifies the default entry point, starts the domain `production` in the foreground so that Payara Server becomes the main process. The _tini_ tool is used to guarantee that the Payara Server process runs seamlessly as the main docker process. The Payara Server process runs under the `payara` user.
+
+It is always recommended to specify the tag name of the Docker Image that you like to run. Otherwise, you are running the latest version and might encounter some unwanted effects due to changes between versions.
+
+....
+docker run -p 8080:8080 payara/server-full:{currentVersion}
+....
+
+
+=== JDK Version
+
+_Since Payara 5.2020.3_
```

Comment:
```suggestion
```
As stated for Payara Micro, we can delete the _Since_ line as all Docker images in Nexus have jdk11 variants.